catalogd/internal/controllers/core/clustercatalog_controller_test.go

```diff
@@ -875,14 +869,13 @@ func TestPollingReconcilerUnpack(t *testing.T) {
 		}
 		return s
 	}
-	successfulStoredCatalogData := func(lastPoll metav1.Time) map[string]storedCatalogData {
+	successfulStoredCatalogData := func(lastPoll time.Time) map[string]storedCatalogData {
 		return map[string]storedCatalogData{
 			"test-catalog": {
 				observedGeneration: successfulObservedGeneration,
-				unpackResult: source.Result{
-					ResolvedSource:            successfulUnpackStatus().ResolvedSource,
-					LastSuccessfulPollAttempt: lastPoll,
-				},
+				ref:                mustRef(t, "my.org/someimage@sha256:"+oldDigest),
+				lastUnpack:         time.Time{},
```

Comment:
These values came from `successfulUnpackStatus().ResolvedSource` (and now I have them hardcoded here). I should check and see if changing them in that function causes this test to fail.

Reply:
Tweaked this code to share the same value used in the function it originally came from.